backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -58,21 +52,30 @@
    */
   private static final List<String> SCOPES = ImmutableList.of(GmailScopes.MAIL_GOOGLE_COM);
 
-  private static final String CREDENTIALS_FILE_PATH = "credentials.json";
+  private static final String CREDENTIALS_FILE_PATH =
+      "/home/grantjustice/IdeaProjects/Ad-lib/backend/credentials.json";
 
   /** The email to be notified */
   private final String toEmail;
 
   /** The notification recipient */
   private final String recipientName;
 
+  /** The gmail dependency */
+  private final Gmail service;
+
   /**
    * @param recipientName Name of the recipient of the user
-   * @param toEmail The email addresse that this email is going to be sent to.
+   * @param toEmail The email address that this email is going to be sent to.
    */
-  public EmailNotifier(String recipientName, String toEmail) {
+  @Inject
+  public EmailNotifier(String recipientName, String toEmail, Gmail gmail) {
+    //    if (gmail == null) {
+    //      throw new InvalidParameterException("Gmail Cannot be Null");
+    //    }
```

Comment:
delete unused? I think it's ok for this class to throw NPEs if gmail is invalid because a caller shouldn't be passing null to the constructor here.

Reply:
Deleted

Reply:
Forgot to push the change? It's still here